.phpstan-baseline.php

```diff
@@ -3712,7 +3712,13 @@
 $ignoreErrors[] = [
 	'message' => '#^Call to function is_array\\(\\) with array will always evaluate to true\\.$#',
 	'identifier' => 'function.alreadyNarrowedType',
-	'count' => 3,
+	'count' => 2,
+	'path' => __DIR__ . '/src/Migration.php',
+];
+$ignoreErrors[] = [
```

Comment:
We should not add new entries in baseline

Reply:
I agree. IMHO, you can remove the `if (is_array($options) && count($options)) {` check. If options is something else than an array, as a developer, I would prefer to have a proper error rather than having my argument silently ignored.

Reply:
I don't want to update code in these kind of PR.